bgs/bgs.go

```diff
@@ -372,6 +372,19 @@ func (bgs *BGS) handleFedEvent(ctx context.Context, host *models.PDS, env *event
 			}
 		}
 
+		return nil
+	case env.RepoHandle != nil:
```

Comment:
If we handle these the same for now, it could be a single compound `env.RepoHandle != nil || env.RepoMigrate != nil`.

I think you should at least check and log whether the account changed due to the event (eg, log "handle was updated due to event" or "got event, but DID doc didn't indicate handle update", along with the expected and found handles).

Reply:
yeah, we could, but it requires adding the whole set of conditionals for figuring out which struct to pull the DID from, which increases the code length anyways.

Reply:
Ah, yeah, sorry for conflicting requests. I think the logging is more important.